specification/cognitiveservices/data-plane/Face/readme.md

```diff
@@ -27,7 +27,7 @@ csharp:
   license-header: MICROSOFT_MIT_NO_VERSION
   azure-arm: false
   namespace: Microsoft.Azure.CognitiveServices.Vision.Face
-  output-folder: $(csharp-sdks-folder)/CognitiveServices/dataPlane/Vision/Microsoft.CognitiveServices.Vision/Generated/Face
+  output-folder: $(csharp-sdks-folder)/CognitiveServices/dataPlane/Vision/Vision/Generated/Face
```

Comment:
did you mean to have `Vision/Vision` in the path?

Reply:
Yes, first Vision is the solution folder, second Vision is the project folder. Having full namespace/package name in the path will push path length limitation.

Reply:
👍